tests/functional/test_custom_task_templates.py

```diff
@@ -378,7 +378,7 @@ def test_custom_task_templates(
                 # Set the target type in the metadata file sent to DataRobot to the correct type.
                 metadata = yaml.safe_load(open(metadata_filename))
                 metadata["targetType"] = target_type
-                yaml.dump(metadata, open(metadata_filename, "w"))
+                yaml.dump(metadata, open(metadata_filename, "w"), default_flow_style=False)
```

Comment:
what's this change doing?

Reply:
This is part of https://github.com/datarobot/datarobot-user-models/commit/2b2b111c2a9dd0c1a1e4c1fa858a563b0f69b803
introduced after 1.6.2 release.
Another part of this PR I backported manually before.